schema_utils_test.py

```diff
@@ -210,7 +218,15 @@ def _validate_ui_config(obj_type, ui_config):
 
 
 def _validate_validator(obj_type, validator):
-    """Validates the value of a 'validator' field."""
+    """Validates the value of a 'validator' field.
+
+    Args:
+        obj_type: str. UI config spec type.
```

Comment:
`The type of the object.` right? (I'm finding it hard to understand `UI config spec type.` can you please explain?

Reply:
There was a similar validation function for UI spec in the file. I had copied the docstring which I wrote for that since the structure is the same. I changed the definition of validator but forgot to change the definition of the object type, Sorry.